scalableminds/webknossos
# Duplicate datasets

Mailable description of changes (needs to be understandable by webknossos mailing list people):
- Fixes datasets being added multiple times

Steps to test:
- symlink a dataset from another team
- start wK
- see, that the dataset exists only once in the DB

Issues:
- fixes #1677 

------
